src/_locales/de/messages.json

```diff
@@ -31,30 +31,6 @@
         "message": "Erkennen von unbekannten Titeln erzwingen",
         "description": "Option title"
     },
-    "optionScrobblePodcasts": {
-        "message": "Scrobble podcasts",
-        "description": "Option label"
-    },
-    "optionScrobblePodcastsTitle": {
-        "message": "Scrobble podcast episodes",
-        "description": "Option title"
-    },
-    "optionsHidden": {
-        "message": "Versteckt",
-        "description": "'Hidden' section"
-    },
-    "optionScrobblePercent": {
-        "message": "Scrobble bei",
-        "description": "Option title"
-    },
-    "optionScrobblePercentSuffix": {
```

Comment:
Wonder why some of these are gone :/

Reply:
Same reason as the english ones i think, directly committed at some point and not in transifex

Reply:
Ahh it's not in the source - I guess someone modified manually - updated on transfix